src/Compilers/CSharp/Test/Syntax/Syntax/FieldAndValueKeywordTests.cs

```diff
@@ -561,6 +582,7 @@ class C
             [CombinatorialValues(LanguageVersion.CSharp12, LanguageVersion.Preview)] LanguageVersion languageVersion)
         {
             string source = """
+                #pragma warning disable 168
```

Comment:
Please include something like the message of the diagnostic which is being disabled or its name in ErrorCode to hint at what is going on here.